agent/provider/src/startup_config.rs

```diff
@@ -60,6 +60,11 @@ pub struct ProviderConfig {
         env = "YA_RT_STORAGE")
     ]
     pub rt_storage: Option<f64>,
+
+    /// Negotiator strategy used by this provider.
+    /// Can be one of: LimitAgreements, AcceptAll.
+    #[structopt(long, default_value = "LimitAgreements")]
+    pub negotiator_strategy: String,
```

Comment:
pls define a dedicated Enum with possible strategies

Reply:
`AcceptAll` doesn't seem to be a viable strategy with the current implementation

Reply:
@mfranciszkiewicz oh, how come?

Reply:
@kubkon `AcceptAllNegotiator` accepts every received proposal / agreement, so it can potentially spawn an unlimited number of activities. Providers don't currently manage free hardware resources

Reply:
Makes sense! In that case, I'm gonna go ahead and close this PR for the moment.